presto-docs/src/main/sphinx/release/release-0.241.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.241
+=============
+
+General Changes
+_______________
+* Fix incorrect results from function classification_precision() introduced in release 0.239
+* Add dynamic filtering and bucket pruning support for inner join and semi join, which can avoid full table scan on the probe side of broadcast joins or collocated joins when the build side is small. This can be enabled with the ``experimental.enable-dynamic-filtering`` configuration property and ``enable_dynamic_filtering`` system session property. The build side join key space size can be tuned by the ``experimental.dynamic-filtering-max-per-driver-row-count`` and ``experimental.dynamic-filtering-max-per-driver-size`` configuration properties and ``dynamic_filtering_max_per_driver_row_count`` and ``dynamic_filtering_max_per_driver_size`` session properties. Currently, only Hive connector can benefit from the feature.
```

Comment:
1. this item is a performance improve, so the line should starts with "Improve".
```
Improve performance of broadcast and collocated join by supporting dynamic filtering and bucket pruning for inner and semi joins.
```
2. the item is way too long. Remove every after "The build side join key ...", and add a link to the PR or the issue instead. Something like
```
(:pr:`12345`)
```

Reply:
> Improve performance of broadcast and collocated join by supporting dynamic filtering and bucket pruning for inner and semi joins.

@caithagoras thanks for the rewriting. But this PR's primary goal is not improving the join's performance but the scan performance on the probe side. So I changed it to `Improve performance for queries with broadcast or collocated joins by adding dynamic filtering and bucket pruning support. `